core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -302,7 +302,7 @@ public void waitUntilKafkaReaderReachesLastOffset(String subject, int timeoutMs)
   private void waitUntilKafkaReaderReachesOffset(long offset, int timeoutMs) throws StoreException {
     log.info("Wait to catch up until the offset at " + offset);
     kafkaTopicReader.waitUntilOffset(offset, timeoutMs, TimeUnit.MILLISECONDS);
-    log.debug("Reached offset at " + offset);
+    log.info("Reached offset at " + offset);
```

Comment:
Can we parameterize this instead of building the string ? We do that pretty consistently elsewhere.

Reply:
Thanks, fixed